Legislative Insight.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "2bedae3c-bab5-447f-b127-e9babc0e9cfe",
 	"label": "Legislative Insight",
 	"creator": "Kari Hemdal",
-	"target": "^https?://www\\.(?:conquest-leg-insight-cert|conquest-leg-insight)\\.com/legislativeinsight/LegHistMain\\.jsp",
+	"target": "^https?://((li.proquest|preprod.li.proquest)\.com/legislativeinsight/LegHistMain\.jsp)",
```

Comment:
The double backslashes were correct. `^https?://(preprod\\.)?li\\.proquest\\.com/legislativeinsight/LegHistMain\\.jsp` is what you want.